lib/gs-plugin-vfuncs.h

```diff
@@ -623,6 +623,26 @@ gboolean	 gs_plugin_app_copy			(GsPlugin	*plugin,
 							 GCancellable	*cancellable,
 							 GError		**error);
 
+/**
+ * gs_plugin_os_get_copyable:
+ * @plugin: a #GsPlugin
+ * @copy_dest: an absolute path of the destination directory
+ * @copyable: whether this app can be copied
+ * @cancellable: a #GCancellable, or %NULL
+ * @error: a #GError, or %NULL
+ *
+ * Determine whether gs_plugin_os_copy() with this @app and @copy_dest should
+ * succeed (barring runtime errors which cannot be known in advance).
+ *
+ * Returns: %FALSE if copying the OS to @copy_dest will certainly fail; %TRUE
+ * otherwise
```

Comment:
How does the return value differ from the value returned in `copyable`?

Reply:
From the implementations, it seems like the return value actually corresponds to `*error == NULL`.

Reply:
`copyable` is independent of the return value. The return value of the function does match `*error == NULL` as all the other similar functions. It's redundant with `*error` for convenience (so you can put it in an `if` statement and in case you don't care about the actual `error`). This function signature pattern is also extremely common in `g-s` so I don't think I should break it.

Reply:
Documentation needs updating then, I think.